README.md

```diff
@@ -1,8 +1,8 @@
-# Heimdall Enterprise Server 2.0
+# Heimdall
```

Comment:
I would keep the version, example: Heimdall V2.x

Reply:
I think it is better to leave it without because we won't have to update it so often and because people can see the latest release number on the "Releases" or "Tags"